staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -81,9 +81,12 @@ type Options struct {
 	// See: https://openid.net/specs/openid-connect-core-1_0.html#IDToken
 	ClientID string
 
-	// PEM encoded root certificate contents of the provider.
+	// PEM encoded root certificate contents of the provider.  Ignored when Client is not nil.
```

Comment:
would it be better to make this and `Client` mutually exclusive?

Reply:
Yeah I think so.  Updated and added unit test.